locales/en/messages.json

```diff
@@ -3726,6 +3726,12 @@
     "pidTuningDynamicNotchMaxHz": {
         "message": "Dynamic Notch Max Hz"
```

Comment:
Are we absolutely sure we don't want "Dynamic Notch Max Frequency [Hz]" and "Dynamic Notch Min Frequency [Hz]?

Reply:
True, we must continue with the same style than others elements on the tab.

Reply:
Okey @klutvott123 i have change also for this 2 forgotten parameters.